src/Widgets/CalendarButton.vala

```diff
@@ -118,25 +93,22 @@ public class Maya.View.Widgets.CalendarButton : Gtk.ToggleButton {
 
         list_box.row_activated.connect ((row) => {
             current_source = ((CalendarGrid)row.get_child ()).source;
+            popover.popdown ();
```

Comment:
unsure that you really want this

Reply:
Yeah to really mimic the expected combobox behavior we want the popover to hide after we make a selection